cup_of_join.py

```diff
@@ -0,0 +1,20 @@
+def join(*lists ,sep='-'):
```

Comment:
read about type hints  and return types, and add them to your functions - https://docs.python.org/3/library/typing.html

Reply:
![Capture](https://user-images.githubusercontent.com/13467749/160142235-1a2e411f-b9f0-4c29-a499-67dc3739f680.JPG)